txmongo/collection.py

```diff
@@ -2,7 +2,7 @@
 # Use of this source code is governed by the Apache License that can be
 # found in the LICENSE file.
 
-import types
+from __future__ import absolute_import, division
```

Comment:
Ok, I don't mind at all. Just thought this is a trace of some changes that were reverted.